test/lib/jdk/test/lib/Platform.java

```diff
@@ -358,7 +364,10 @@ public static Path jvmLibDir() {
     }
 
     private static String variant() {
-        if (Platform.isServer()) {
+        if (Platform.isServer()
```

Comment:
Might it be a better solution to modify `isServer()`? I think tests will expect at least one of the `isXXX()` methods involving `vmName` to return true.

Reply:
Yes, we will be able to support more tests. Let me draft a new PR.

Reply:
That will break isDefaultCDSArchiveSupported() so take that into account.

Reply:
Created https://github.com/ibmruntimes/openj9-openjdk-jdk/pull/391.